test/cmdlineTests/storage_layout_transient_value_types/input.json

```diff
@@ -0,0 +1,16 @@
+{
+	"language": "Solidity",
+	"sources": {
+		"fileA": {
+			"content": "//SPDX-License-Identifier: GPL-3.0\npragma solidity >=0.0;\ncontract A { uint transient x; bytes32 transient b; address transient addr; }"
```

Comment:
Oh, also, I think we should check what the SMTChecker will do in presence of `transient` storage variables, because I expect it will not simply fail, but just confidently tell you something that's wrong and this is dangerous.

Take this contract for example:
```solidity
contract C {
    uint /* transient */ x = 42;

    function test() public view { assert(x == 42); }
}
```
```bash
solc test.sol --model-checker-engine all
```
```
Info: CHC: 1 verification condition(s) proved safe! Enable the model checker option "show proved safe" to see all of them.
```

If you get the same answer after uncommenting `transient`, we need to stick an unimplemented assert somewhere in it to make sure that it will just fail when you try to use it with transient variables

Reply:
This is addressed by https://github.com/ethereum/solidity/pull/15001/commits/de95825a4f77b5db7b710c3fc053e983a21fcdd5

Reply:
You should add my example as a SMTChecker test so that we can see that it works.

Reply:
Added.